pkg/Microsoft.Private.CoreFx.NETCoreApp/netcoreapp.rids.props

```diff
@@ -1,85 +1,26 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <Choose>
-    <When Condition="'$(PackageRID)' != ''" />
-    <When Condition="'$(_runtimeOSFamily)' == 'win'">
-      <PropertyGroup>
-        <PackageRID Condition="'$(ArchGroup)' == 'x86' OR '$(ArchGroup)' == 'x64'">win7-$(ArchGroup)</PackageRID>
-        <PackageRID Condition="'$(ArchGroup)' == 'arm'">win8-$(ArchGroup)</PackageRID>
-        <PackageRID Condition="'$(ArchGroup)' == 'arm64'">win10-$(ArchGroup)</PackageRID>
-        <PackageRID Condition="'$(PortableBuild)' == 'true'">win-$(ArchGroup)</PackageRID>
-      </PropertyGroup>
-    </When>
-    <When Condition="'$(_runtimeOSFamily)' == 'osx'">
-      <PropertyGroup>
-        <PackageRID>osx.10.12-$(ArchGroup)</PackageRID>
-        <PackageRID Condition="'$(PortableBuild)' == 'true'">osx-$(ArchGroup)</PackageRID>
-      </PropertyGroup>
-    </When>
-    <When Condition="'$(_runtimeOSFamily)' == 'rhel'">
-      <PropertyGroup>
-        <PackageRID>rhel.7-$(ArchGroup)</PackageRID>
-        <PackageRID Condition="'$(PortableBuild)' == 'true'">linux-$(ArchGroup)</PackageRID>
-      </PropertyGroup>
-    </When>
-    <Otherwise>
-      <PropertyGroup>
-        <PackageRID>$(RuntimeOS)-$(ArchGroup)</PackageRID>
-        <PackageRID Condition="'$(PortableBuild)' == 'true'">linux-$(ArchGroup)</PackageRID>
-      </PropertyGroup>
-    </Otherwise>
-  </Choose>
+  <PropertyGroup>
+    <PackageRID>win-$(ArchGroup)</PackageRID>
+    <PackageRID Condition="'$(OSGroup)' == 'OSX'">osx-$(ArchGroup)</PackageRID>
+    <PackageRID Condition="'$(OSGroup)' == 'Linux'">linux-$(ArchGroup)</PackageRID>
```

Comment:
I simplified this more by using _runtimeOSFamily which we compute by removing any version information from RuntimeOS property.